### ISSUE TITLE
Alternative to deleting mods to be updated

### ISSUE THREAD

**Issue by Evolution0** [Dynious/VersionChecker#68]
Currently if you double-click a mod to update it that doesn't support automatically updating and instead opt to not download it at that time.. it gets deleted.

Now imagine doing this with 20+ mods only to realize you don't remember *which* needed to be updated and now need to somehow figure out which were removed.

Maybe append .disabled to the filename of the mod instead? Or move it to another directory?

Appending .disabled to them would allow you to keep them where they are but have them display as disabled in MultiMC which is nice. (and users can simply "remove" in MultiMC or delete them manually)

It may seem a little silly but I seriously did not expect it to be so heavy handed and simply delete them. I had to check my fml log to see which mods loaded on start and get them back.

**Comment by Dynious** [comment]
Hmm, did you maybe press the 'Mark as downloaded' button'? When mods that cannot be automatically updated a button is added that will allow user to mark mods as downloaded which will delete the old mod files so that they don't have to manually delete them all.

I found out that the is a small chance when double-clicking an entry a button can be pressed in the pop-up menu. I now added in some protection to make sure this can't happen anymore.
